core/error/error_macros.h

```diff
@@ -30,12 +30,12 @@
 
 #pragma once
 
-#include "core/object/object_id.h"
 #include "core/typedefs.h"
 
-#include <atomic> // We'd normally use safe_refcount.h, but that would cause circular includes.
+#include <atomic> // IWYU pragma: keep // We'd normally use `safe_refcount.h`, but that would cause circular includes.
```

Comment:
Why does clangd want to remove this one? It's clearly used in this file.

Reply:
Ah it doesn't deal with macros. Could have "Used in macro." comment like other cases in this PR.